### PR TITLE
[do not merge] Chore/dot com prep

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

updates all link/urls to be relative.
this is to support our upcoming change to .com

markdown files have all references to .io supabase sites changed to .com

### CODE REVIEW

File: web/docs/guides/auth/auth-twilio.mdx [hunk 1]
```diff
@@ -296,5 +296,5 @@ The user does not have a password therefore will need to sign in via this method
 ## Resources
 
 - [Twilio Signup](https://www.twilio.com/try-twilio)
-- [Supabase Dashboard](https://app.supabase.io)
-- [Supabase Row Level Security](https://supabase.io/docs/guides/auth#row-level-security)
+- [Supabase Dashboard](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/auth/auth-twilio.mdx:299]: this one will stay .io right?

File: web/docs/guides/client-libraries.mdx [hunk 2]
```diff
@@ -241,17 +224,14 @@ Since Postgres is a Relational database, the client makes it simple to query tab
 <TabItem value="js">
 
 ```js
-const { data, error } = await supabase
-  .from('countries')
-  .select(`
+const { data, error } = await supabase.from('countries').select(`
```
Comment (kiwicopple) [inline web/docs/guides/client-libraries.mdx:227]: can this stay the same?

File: web/docs/guides/client-libraries.mdx [hunk 3]
```diff
@@ -356,15 +328,12 @@ You can create data easily using [`insert()`](/docs/reference/javascript/insert)
 <TabItem value="js">
 
 ```js
-const { data, error } = await supabase
-  .from('cities')
-  .insert([
-    { name: 'The Shire', country_id: 554 },
-    { name: 'Rohan', country_id: 555 },
-  ])
+const { data, error } = await supabase.from('cities').insert([
```
Comment (kiwicopple) [inline web/docs/guides/client-libraries.mdx:331]: also less readable

File: web/docs/guides/client-libraries.mdx [hunk 4]
```diff
@@ -517,9 +477,7 @@ void main() async {
 
 </Tabs>
 
-
-
 ## Next steps
 
 - View the [Client Docs](/docs/reference/javascript/supabase-client)
-- Sign in: [app.supabase.io](https://app.supabase.io)
+- Sign in: [app.supabase.com](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/client-libraries.mdx:483]: .io

File: web/docs/guides/database.mdx [hunk 5]
```diff
@@ -197,5 +190,4 @@ Read more in [Database Configuration](/docs/guides/database/managing-timezones).
 ## Next steps
 
 - Read more about [Postgres](/docs/postgres/server/about)
-- Sign in: [app.supabase.io](https://app.supabase.io)
-
+- Sign in: [app.supabase.com](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/database.mdx:193]: .io

File: web/docs/guides/hosting/docker.mdx [hunk 6]
```diff
@@ -88,4 +88,4 @@ See the following guides to deploy Docker Compose setup using your preferred too
 ## Next steps
 
 - Got a question? [Ask here](https://github.com/supabase/supabase/discussions).
-- Sign in: [app.supabase.io](https://app.supabase.io)
+- Sign in: [app.supabase.com](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/hosting/docker.mdx:91]: .io

File: web/docs/guides/hosting/overview.mdx [hunk 7]
```diff
@@ -10,35 +10,33 @@ import JwtGenerator from '@site/src/components/JwtGenerator'
 
 There are several ways to use Supabase:
 
-- [Supabase Cloud](https://app.supabase.io): you don't need to deploy anything. We will manage and scale your infrastructure. 
+- [Supabase Cloud](https://app.supabase.com): you don't need to deploy anything. We will manage and scale your infrastructure.
```
Comment (kiwicopple) [inline web/docs/guides/hosting/overview.mdx:13]: .io - also we should change this to "Supabase Platform"

File: web/docs/guides/hosting/platform.mdx [hunk 8]
```diff
@@ -4,10 +4,9 @@ title: Supabase Platform
 description: Getting started with the Supabase platform.
 ---
 
-
 Supabase is a hosted platform which makes it very simple to get started without needing to manage any infrastructure.
 
-Visit [app.supabase.io](https://app.supabase.io) and sign in to start creating projects.
+Visit [app.supabase.com](https://app.supabase.com) and sign in to start creating projects.
```
Comment (kiwicopple) [inline web/docs/guides/hosting/platform.mdx:9]: .io

File: web/docs/guides/hosting/platform.mdx [hunk 9]
```diff
@@ -18,18 +17,18 @@ Organizations are a way to group your projects. Every Organization can be config
...
 - A dedicated Postgres database. [Learn more](/docs/guides/database)
-- Auto-generated APIs. 
+- Auto-generated APIs.
 - User management. [Learn more](/docs/guides/auth)
 - Storage. [Learn more](/docs/guides/storage)
 
 ## Next Steps
 
-- Sign in to [app.supabase.io](https://app.supabase.io)
+- Sign in to [app.supabase.com](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/hosting/platform.mdx:34]: .io

File: web/docs/guides/with-nextjs.mdx [hunk 10]
```diff
@@ -44,20 +44,19 @@ If you want to do it yourself, let's get started: -->
...
 
-Before we start building we're going to set up our Database and API. This is as simple as starting a new Project in Supabase 
+Before we start building we're going to set up our Database and API. This is as simple as starting a new Project in Supabase
 and then creating a "schema" inside the database.
 
-### Create a project 
+### Create a project
 
-1. Go to [app.supabase.io](https://app.supabase.io).
+1. Go to [app.supabase.com](https://app.supabase.com).
```
Comment (kiwicopple) [inline web/docs/guides/with-nextjs.mdx:52]: .io

File: web/docs/guides/with-nextjs.mdx [hunk 11]
```diff
@@ -549,12 +537,11 @@ return (
...
-
 ```
 
 ## Next steps
 
 At this stage you have a fully functional application!
 
 - Got a question? [Ask here](https://github.com/supabase/supabase/discussions).
-- Sign in: [app.supabase.io](https://app.supabase.io)
+- Sign in: [app.supabase.com](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/with-nextjs.mdx:547]: io

File: web/docs/guides/with-react.mdx [hunk 12]
```diff
@@ -44,20 +44,19 @@ If you want to do it yourself, let's get started: -->
...
 
-Before we start building we're going to set up our Database and API. This is as simple as starting a new Project in Supabase 
+Before we start building we're going to set up our Database and API. This is as simple as starting a new Project in Supabase
 and then creating a "schema" inside the database.
 
-### Create a project 
+### Create a project
 
-1. Go to [app.supabase.io](https://app.supabase.io).
+1. Go to [app.supabase.com](https://app.supabase.com).
```
Comment (kiwicopple) [inline web/docs/guides/with-react.mdx:52]: io

File: web/docs/guides/with-react.mdx [hunk 13]
```diff
@@ -470,9 +463,7 @@ export default function Avatar({ url, size, onUpload }) {
       const fileName = `${Math.random()}.${fileExt}`
       const filePath = `${fileName}`
 
-      let { error: uploadError } = await supabase.storage
-        .from('avatars')
-        .upload(filePath, file)
+      let { error: uploadError } = await supabase.storage.from('avatars').upload(filePath, file)
```
Comment (kiwicopple) [inline web/docs/guides/with-react.mdx:466]: keep the same?

File: web/docs/guides/with-react.mdx [hunk 14]
```diff
@@ -545,12 +534,11 @@ return (
...
-
 ```
 
 ## Next steps
 
 At this stage you have a fully functional application!
 
 - Got a question? [Ask here](https://github.com/supabase/supabase/discussions).
-- Sign in: [app.supabase.io](https://app.supabase.io)
+- Sign in: [app.supabase.com](https://app.supabase.com)
```
Comment (kiwicopple) [inline web/docs/guides/with-react.mdx:544]: .io

File: web/docs/guides/client-libraries.mdx (hunk 2 above)
Comment (MildTomato) [reply web/docs/guides/client-libraries.mdx:227]: ahh damn its my prettier

**Pull request closed without merging**